content/docs/11.migration-guide/0.23.0/default-pull-policy.md

```diff
@@ -0,0 +1,32 @@
+---
+title: The default pullPolicy for Docker-based tasks changed
+icon: /docs/icons/migration-guide.svg
+release: 0.23.0
+editions: ["OSS", "EE"]
+---
+
+## Overview
+
+Due to the new [Docker Hub pull usage and limits](https://docs.docker.com/docker-hub/usage/pulls/), all the Docker-based Kestra tasks will have their default `pullPolicy` updated from `ALWAYS` to `IF_NOT_PRESENT` to avoid any pull limit issue. Read more about the change in the [GitHub issue](https://github.com/kestra-io/plugin-scripts/issues/230).
+
+Previously, the following flow would have the `pullPolicy` default to `ALWAYS`:
+
+```yaml
+id: docker_script_runner
+namespace: company.team
+
+tasks:
+  - id: shell
+    type: io.kestra.plugin.scripts.shell.Commands
+    containerImage: centos
+    taskRunner:
+      type: io.kestra.plugin.scripts.runner.docker.Docker
+      cpu:
+        cpus: 1
+    commands:
+      - echo "Hello World!"
+```
+
+Now, the plugin defaults to `IF_NOT_PRESENT`.
```

Comment:
The plugin-docker's `Run` `pullPolicy` property still defaults to `ALWAYS`: https://github.com/kestra-io/plugin-docker/blob/d27111c783112fb2526732e1d3f47102964cd7d2/src/main/java/io/kestra/plugin/docker/Run.java#L190. Maybe we should update it to `IF_NOT_PRESENT` too.